db/fixtures/miq_event_definitions.csv

```diff
@@ -106,7 +106,7 @@ vm_provisioned,VM Provision Complete,Default,vm_process
 vm_start,VM Power On,Default,vm_operational
 request_vm_start,VM Power On Request,Default,vm_operational
 # vm_shutdown,VM Shutdown,Default,vm_operational
-# vm_resume,VM Resume,Default,vm_operational
+vm_resume,VM Resume,Default,vm_operational
```

Comment:
@gmcculloug This [one](https://github.com/ManageIQ/manageiq/blob/bee0cfc3cc4132585cc02922d2733e64119b72e1/app/models/manageiq/providers/microsoft/infra_manager.rb#L99) might be a candidate. 
But so far there is not any event in automate model for Microsoft. Is it right?